vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -1011,8 +1009,15 @@ EXPORT(int, sceGxmPrecomputedDrawSetParamsInstanced) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceGxmPrecomputedDrawSetVertexStream) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceGxmPrecomputedDrawSetVertexStream, SceGxmPrecomputedDraw *precomputedDraw, uint32_t streamIndex, Ptr<const void> streamData) {
+    if (precomputedDraw == nullptr || streamData.get(host.mem) == nullptr)
```

Comment:
if (precomputedDraw == nullptr || streamData) {

Reply:
if (!precomputedDraw || !streamData) { I meant

Reply:
ah yes sorry, old code, i have forget update it